src/comm/aca_comm_mgr.cpp

```diff
@@ -246,7 +246,13 @@ int Aca_Comm_Manager::update_goal_state(GoalStateV2 &goal_state_message,
                 port_operation_time, us_to_ms(port_operation_time),
                 neighbor_operation_time, us_to_ms(neighbor_operation_time),
                 dhcp_operation_time, us_to_ms(dhcp_operation_time));
-
+*/
```

Comment:
if it doesn't impact performance as I remember, we can keep it

Reply:
Done